packages/utils/spatial-utils/src/spatial.js

```diff
@@ -273,27 +273,30 @@ export function isInterleaved(shape) {
  * @returns {object[]} An array of selected channels with default
  * domain/slider settings.
  */
-export async function initializeLayerChannels(loader, use3d) {
+export async function initializeLayerChannels(loader, use3d, isBitmask) {
   const result = [];
   const source = getSourceFromLoader(loader);
   // Add channel automatically as the first avaialable value for each dimension.
   let defaultSelection = buildDefaultSelection(source);
   defaultSelection = isInterleaved(source.shape)
     ? [{ ...defaultSelection[0], c: 0 }] : defaultSelection;
-  const stats = await getMultiSelectionStats({
-    loader: loader.data, selections: defaultSelection, use3d,
-  });
-
-  const domains = isRgb(loader, null)
-    ? [[0, 255], [0, 255], [0, 255]]
-    : stats.domains;
-  const colors = isRgb(loader, null)
-    ? [[255, 0, 0], [0, 255, 0], [0, 0, 255]]
-    : null;
-  const sliders = isRgb(loader, null)
-    ? [[0, 255], [0, 255], [0, 255]]
-    : stats.sliders;
-
+  let domains = defaultSelection.map(i => [0, 255])
+  let sliders = defaultSelection.map(i => [0, 255])
+  let colors = defaultSelection.map((_, i) => PALETTE[i])
```

Comment:
Not sure of a better way to do this - maybe if-then?  But the operation is expensive.  I think the first two defaults don't matter anyway, and the `colors` seems reasonable but not sure of the default palette for this.